content/en/docs/ops/configuration/mesh/config-resource-ready/snips.sh

```diff
@@ -21,7 +21,7 @@
 ####################################################################################################
 
 snip_install_with_enable_status() {
-istioctl install --set values.pilot.env.PILOT_ENABLE_STATUS=true --set values.pilot.env.PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING=true --set values.global.istiod.enableAnalysis=true
+istioctl install --set values.pilot.env.PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING=true --set values.pilot.env.PILOT_ENABLE_STATUS=true --set values.pilot.env.PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING=true --set values.global.istiod.enableAnalysis=true
```

Comment:
This isn't ideal, since the test is already setting it. Since they both are set to true, it still works. You could make the snip generator a little smarter, by not doing the replace if the command already sets values.pilot.env.PILOT_ENABLE_CONFIG_DISTRIBUTION_TRACKING, but it's not really necessary.

Reply:
I did try and code this up with anther match, but it didn't work on the first pass. And since it was setting it to true, we should be OK as mentioned.